src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: content-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 50px;
+}
+
+img {
+  display: flex;
+  align-self: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  justify-content: center;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__link {
+
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, serif;
+  line-height: 14px;
+  padding: 23px 0;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;
+}
+
+.nav__list li:not(:first-child) {
+  margin-left: 10px;
+}
+
+.nav__list li:not(:last-child) {
+  margin-right: 10px;
+}
```

Comment:
It's not okay. Try to find a more straightforward way for this logic. You should set margins only from one side, not for both.